README.md

```diff
@@ -1,6 +1,4 @@
-# loopback-connector-couchbase3
-
-[![Build Status](https://travis-ci.org/Wiredcraft/loopback-connector-couchbase3.svg)](https://travis-ci.org/Wiredcraft/loopback-connector-couchbase3) [![Coverage Status](https://coveralls.io/repos/Wiredcraft/loopback-connector-couchbase3/badge.svg?branch=master&service=github)](https://coveralls.io/github/Wiredcraft/loopback-connector-couchbase3?branch=master)
```

Comment:
Are you removing the CI badge intentionally?

Reply:
The flag would be broken once we changed repo's name I assumed.

Reply:
sure, created a new ticket to follow up #90